invenio_search_ui/config.py

```diff
@@ -10,6 +10,8 @@
 
 from __future__ import absolute_import, print_function
 
+APP_THEME = ['semantic-ui', 'bootstrap3']
```

Comment:
Comment: I guess this is also just for testing?

Reply:
I left it until I'm sure where this should go, cookiecutter right?